### PR TITLE
Improves Spanish Translation

### PR DESCRIPTION
**Pull request by GianCastle**
**What this PR does?**
Improvements on Spanish translations

**Why are you adding given library? (Short description why do you think that this library is important)**
- Typescript as a required topic. Typescript still growing and growing every single year, so React projects with Typescript codebases is growing too.

### CODE REVIEW

**Review by adam-golab — APPROVED** [review]
Looks good to me, but it would be great if someone else who knows Spanish could take a look at this.
Thanks for contribution :)

File: README-ES.md [hunk 1]
```diff
@@ -12,12 +12,13 @@
 
 [README in Russian](README-RU.md)
 
-> Roadmap para convertirse en un desarrollador React en 2019::
+> Roadmap para convertirse en un desarrollador React en 2019:
 
-A continuación puede encontrar un gráfico que muestra las rutas que puede tomar y las bibliotecas que le gustaría aprender para convertirse en un desarrollador React. Hice esta tabla como una sugerencia para todos los que me preguntan: "¿Qué debo aprender a continuación como desarrollador de React?"
+A continuación, podrás encontrar un gráfico que muestra los diferentes caminos que puedes elegir y las librerias que te deberian de interesarte para convertirte en un desarrollador React. Hice esta tabla como una sugerencia para todos los que me preguntan: "¿Qué debo aprender a continuación para aprender React?"
```
Comment (taverasmisael) [inline README-ES.md]:
```suggestion
A continuación, podrás encontrar un gráfico que muestra los diferentes caminos que puedes elegir y las librerías que te deberían de interesarte para convertirte en un desarrollador React. Hice esta tabla como una sugerencia para todos los que me preguntan: "¿Qué debo aprender a continuación para aprender React?"
```

File: README-ES.md [hunk 2]
```diff
@@ -12,12 +12,13 @@
...
 
-## Descargo
-> El propósito de este roadmap es darte una idea sobre el paisaje. Este lo guiará en caso de estár confundido acerca de qué aprender, en lugar de animarlo a elegir lo que es moderno y actual. Debe comprender mejor por qué una herramienta sería más adecuada para algunos casos que la otra y recordar que la moda y la moda no siempre son las más adecuadas para el trabajo
+## Descargo de responsabilidad
+
+   > El propósito de este roadmap es darte una idea sobre el ecosistema. Esto podrá guiarte en caso de que estés confundido sobre qué aprender, en lugar de animarte a elegir lo que es moderno y actual y ya esta. Debes comprender mejor el por qué una herramienta sería más adecuada para algunos casos que otra y recordar que lo que está de moda no siempre es la herramienta más adecuada para el trabajo.
```
Comment (taverasmisael) [inline README-ES.md:21]:
```suggestion
   > El propósito de este roadmap es darte una idea sobre el ecosistema. Esto podrá guiarte en caso de que estés confundido sobre qué aprender, en lugar de animarte a elegir lo que es moderno y actual. Debes comprender mejor el por qué una herramienta sería más adecuada para algunos casos que otra y recordar que lo que está de moda no siempre es la herramienta más adecuada para el trabajo.
```

File: README-ES.md [hunk 3]
```diff
@@ -27,40 +28,41 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
...
+        * Aprender operaciones básicas dentro del DOM
+        * Aprender los mecanismos típicos de JS (Hoisting, Event Bubbling, programación basada en Prototipos)
+        * Hacer algunas llamadas AJAX (XHR)
+        * Aprender nuevas características (ECMAScript 6+)
+        * Además, familiarizate con la libreria jQuery
+2. Conceptos Generales de Programación
+   * Aprende Git, crea algunos repositorios en GitHub, comparte tú código con otras personas
+   * Conoce el protocolo HTTP (S), qué son los métodos de solicitud (GET, POST, PUT, PATCH, DELETE, OPTIONS)
+   * No tengas miedo de usar Google, [Power Searching with Google](http://www.powersearchingwithgoogle.com/)
+   * Familiarizate con la Terminal (Bash, Consola, como quieras llamarlo), configura tú shell (Bash, ZSH, Fish)
```
Comment (taverasmisael) [inline README-ES.md:48]:
```suggestion
   * Familiarizate con la terminal, configura tú shell (Bash, ZSH, Fish)
```

File: README-ES.md [hunk 4]
```diff
@@ -77,16 +79,16 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
         * [OOCSS](https://github.com/stubbornella/oocss/wiki)
         * [SMACSS](https://smacss.com/)
         * [SUITCSS](https://suitcss.github.io/)
-    4. CSS en JS
+    4. CSS en JS (CssInJS)
```
Comment (taverasmisael) [inline README-ES.md:82]:
```suggestion
    4. CSS en JS (CSS in JS)
```

File: README-ES.md [hunk 5]
```diff
@@ -77,16 +79,16 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
...
         * [Styled Components](https://www.styled-components.com/)
         * [Radium](https://formidable.com/open-source/radium/)
         * [Emotion](https://emotion.sh/)
         * [JSS](http://cssinjs.org/)
         * [Aphrodite](https://github.com/Khan/aphrodite)
-6. Manejo del estado
+6. Manejo del estados (State Managers)
```
Comment (taverasmisael) [inline README-ES.md:88]:
```suggestion
6. Manejadores de estado (State Managers)
```

File: README-ES.md [hunk 6]
```diff
@@ -77,16 +79,16 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
...
     1. [Component State](https://reactjs.org/docs/faq-state.html)/[Context API](https://reactjs.org/docs/context.html)
     2. [Redux](https://redux.js.org/)
-        1. Acciones asincrónicas (Efectos secundarios)
+        1. Acciones asincrónicas (Side-Effects, Async Actions)
```
Comment (taverasmisael) [inline README-ES.md:91]:
```suggestion
        1. Acciones asincrónicas (Side-Effects)
```

File: README-ES.md [hunk 7]
```diff
@@ -99,21 +101,21 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
             * [Redux Phoenix](https://github.com/adam-golab/redux-phoenix)
         4. [Redux Form](https://redux-form.com)
     3. [MobX](https://mobx.js.org/)
-7. Comprobadores de tipos
+7. Comprobadores de tipos (Typechecking)
```
Comment (taverasmisael) [inline README-ES.md:104]:
```suggestion
7. Comprobadores de tipos (Type Checkers)
```

File: README-ES.md [hunk 8]
```diff
@@ -130,31 +132,31 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
     * [RxJS](http://reactivex.io/)
     * [ImmutableJS](https://facebook.github.io/immutable-js/)
     * [Ramda](https://ramdajs.com/)
-12. Pruebas
-    1. Pruebas Unitarias
+12. Pruebas (Testing)
```
Comment (taverasmisael) [inline README-ES.md:135]:
```suggestion
12. Pruebas
```

File: README-ES.md [hunk 9]
```diff
@@ -130,31 +132,31 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
...
+    1. Pruebas Unitarias (Unit Testing)
```
Comment (taverasmisael) [inline README-ES.md:136]:
```suggestion
    1. Pruebas Unitarias
```

File: README-ES.md [hunk 10]
```diff
@@ -130,31 +132,31 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
...
         * [Jest](https://facebook.github.io/jest/)
         * [Enzyme](http://airbnb.io/enzyme/)
         * [Sinon](http://sinonjs.org/)
         * [Mocha](https://mochajs.org/)
         * [Chai](http://www.chaijs.com/)
         * [AVA](https://github.com/avajs/ava)
         * [Tape](https://github.com/substack/tape)
-    2. Pruebas de extremo a extremo
+    2. Pruebas de extremo a extremo (End To End Testing, E2E)
```
Comment (taverasmisael) [inline README-ES.md:144]:
```suggestion
    2. Pruebas de extremo a extremo (End To End Testing)
```

File: README-ES.md [hunk 11]
```diff
@@ -130,31 +132,31 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
...
         * [Selenium](https://www.seleniumhq.org/), [Webdriver](http://webdriver.io/)
         * [Cypress](https://cypress.io/)
         * [Puppeteer](https://pptr.dev/)
         * [Cucumber.js](https://github.com/cucumber/cucumber-js)
         * [Nightwatch.js](http://nightwatchjs.org/)
-    3. Pruebas de integración
+    3. Pruebas de Integración (Integration Tests)
```
Comment (taverasmisael) [inline README-ES.md:150]:
```suggestion
    3. Pruebas de Integración
```

File: README-ES.md [hunk 12]
```diff
@@ -130,31 +132,31 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
...
         * [Karma](https://karma-runner.github.io/)
-13. Internacionalización
+13. Internacionalización (Int, i18)
```
Comment (taverasmisael) [inline README-ES.md:152]:
```suggestion
13. Internacionalización
```

Or 

```suggestion
13. Internacionalización (i18n)
```

`Int` may convey a data type for the reader and not Internationalization.

File: README-ES.md [hunk 13]
```diff
@@ -170,16 +172,15 @@ A continuación puede encontrar un gráfico que muestra las rutas que puede toma
...
+Si crees que se puede mejorar este roadmap, abre un PR con cualquier actualización y envia cualquier problema. Además, continuaré mejorandolo, por lo que es posible que desee darle una estrella a este repositorio para volver a visitarlo.
 
 ## Contribución
 
-The roadmap is built using [Draw.io](https://www.draw.io/). Project file can be found at `/src` directory. To modify it, open draw.io, click **Open Existing Diagram** and choose `xml` file with project. It will open the roadmap for you. Update it, upload and update the images in readme and create a PR (export as png).
+El diagrama fue elaborado con [Draw.io](https://www.draw.io/). el archivo del proyecto puedes encontrarlo dentro del directorio `/src`. Para modificarlo, abre draw.io, click en **Open Existing Diagram** y elige la opción `xml` file with project. Una vez hecho esto, el diagrama lo podrás ver dentro del programa. actualizalo, súbelo y actualiza la imagen dentro del  README y crea un PR (recuerda que la imagen es un archivo )
```
Comment (taverasmisael) [inline README-ES.md:179]:
```suggestion
El diagrama fue elaborado con [Draw.io](https://www.draw.io/). el archivo del proyecto puedes encontrarlo dentro del directorio `/src`. Para modificarlo, abre draw.io, click en **Open Existing Diagram** y elige la opción `xml` file with project. Una vez hecho esto, el diagrama lo podrás ver dentro del programa. Actualízalo, súbelo y actualiza la imagen dentro del  README y crea un PR (recuerda que la imagen es un archivo ).
```

**Review by taverasmisael — CHANGES_REQUESTED** [review]
Overall looks good, just some missing accents.

## Two quick notes:

1. Remember, the use of `tú` vs `tu`, re-check the list as I found many misuses.
2. Remember that if the list members are complete sentences, it is best to close each one with a period. (please update where needed)

**Pull request closed without merging**